cblite/llm/Gemini.cc

```diff
@@ -22,25 +22,69 @@ using namespace std;
 using namespace fleece;
 using namespace litecore;
 
-alloc_slice Gemini::run(Value rawSrcPropValue, const string& modelName) {
-    string restBody = format("{\"model\": \"models/%s\", \"content\": {\"parts\": [{\"text\": \"%.*s\"}]}}", modelName.c_str(), SPLAT(rawSrcPropValue.asString()));
+vector<alloc_slice> Gemini::run(const string& modelName, vector<Value> propVec) {
+    int batchSize = 50, docRemainder = propVec.size() % batchSize;
```

Comment:
This should be a const int (50) and case it differently to represent its constness (BATCH_SIZE)